gitlab/clients/_httpxclient.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations
+
```

Comment:
Not sure what others think. But I would prefer the name to be either `_httpx_client` or `_httpx`. And the same idea on the current `_requestsclient`. But others may hate that idea!

Reply:
I don't have any strong feeling to either option.